packages/meditrak-server/src/apiV2/GETHandler/helpers.js

```diff
@@ -42,28 +42,36 @@ export const generateLinkHeader = (resource, pageString, lastPage, originalQuery
   return formatLinkHeader(linkHeader);
 };
 
-export const processColumnSelector = (models, unprocessedColumnSelector, baseRecordType) => {
+export const fullyQualifyColumnSelector = (models, unprocessedColumnSelector, baseRecordType) => {
   const [resource, column] = unprocessedColumnSelector.includes('.')
     ? unprocessedColumnSelector.split('.')
     : [baseRecordType, unprocessedColumnSelector];
-
   const recordType = resourceToRecordType(resource);
-  const model = models.getModelForDatabaseType(recordType);
-  const customSelector = model?.customColumnSelectors?.[column];
```

Comment:
This `customSelector` was previously being applied to both sides of the `as`, where we actually only want it on the left, e.g. `select ST_AsGeoJSON(entity.region) as entity.region` rather than `select ST_AsGeoJSON(entity.region) as ST_AsGeoJSON(entity.region)`